### DISCUSSION TITLE
Dreambooth installation error

### DISCUSSION THREAD

**Discussion by Delicialin** [d8ahazard/sd_dreambooth_extension#1467] [Q&A]
I met the error when I install the Dreambooth extension, the log is as follows:

*** Error running install.py for extension D:\Program Files\StableDiffusion\sd-webui-aki-v4.6.1\extensions\sd_dreambooth_extension.
*** Command: "D:\Program Files\StableDiffusion\sd-webui-aki-v4.6.1\python\python.exe" "D:\Program Files\StableDiffusion\sd-webui-aki-v4.6.1\extensions\sd_dreambooth_extension\install.py"
*** Error code: 1
*** stdout: If submitting an issue on github, please provide the full startup log for debugging purposes.
***
*** Initializing Dreambooth
*** Dreambooth revision: bdc784a86349e6529624cb8e4b3339e5d0f01aee
*** Checking xformers...
*** Checking bitsandbytes...
*** Checking bitsandbytes (Windows)
***
*** stderr: D:\Program Files\StableDiffusion\sd-webui-aki-v4.6.1\python\lib\site-packages\pytorch_lightning\utilities\distributed.py:258: LightningDeprecationWarning: `pytorch_lightning.utilities.distributed.rank_zero_only` has been deprecated in v1.8.1 and will be removed in v2.0.0. You can import it from `pytorch_lightning.utilities` instead.
***   rank_zero_deprecation(
*** [2024-03-08 15:42:16,492][DEBUG][git.cmd] - Popen(['git', 'version'], cwd=D:\Program Files\StableDiffusion\sd-webui-aki-v4.6.1, universal_newlines=False, shell=None, istream=None)
*** [2024-03-08 15:42:16,515][DEBUG][git.cmd] - Popen(['git', 'version'], cwd=D:\Program Files\StableDiffusion\sd-webui-aki-v4.6.1, universal_newlines=False, shell=None, istream=None)
*** [2024-03-08 15:42:16,542][DEBUG][git.cmd] - Popen(['git', 'cat-file', '--batch-check'], cwd=D:\Program Files\StableDiffusion\sd-webui-aki-v4.6.1\extensions\sd_dreambooth_extension, universal_newlines=False, shell=None, istream=<valid stream>)
*** Traceback (most recent call last):
***   File "D:\Program Files\StableDiffusion\sd-webui-aki-v4.6.1\extensions\sd_dreambooth_extension\install.py", line 35, in <module>
***     actual_install()
***   File "D:\Program Files\StableDiffusion\sd-webui-aki-v4.6.1\extensions\sd_dreambooth_extension\postinstall.py", line 59, in actual_install
***     check_bitsandbytes()
***   File "D:\Program Files\StableDiffusion\sd-webui-aki-v4.6.1\extensions\sd_dreambooth_extension\postinstall.py", line 217, in check_bitsandbytes
***     if ";" in venv_path:
*** TypeError: argument of type 'NoneType' is not iterable

At the same time, the Stable Diffusion doesn't show Dreambooth tag.
So how to resolve it?

**I tried:**
- Switching the version of Dreambooth
- Switching the version of Stable Diffusion's core
- Downloading the Dreambooth directly in github and installed in CMD
- Install CUDA and reinstall Dreambooth

**Comment by CatofTang** [comment]
It seems that your version of pytorch-lightning is too old. Try 1.9.4 or reinstall requirements list in requirements.txt.
